religion-in-the-roman-empire.csl

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB" name-delimiter="; " names-delimiter="; " delimiter-precedes-last="always" page-range-format="expanded">
+  <info>
+    <title>Religion in the Roman Empire</title>
+    <title-short>RRE</title-short>
+    <id>http://www.zotero.org/styles/religion-in-the-roman-empire</id>
+    <link href="http://www.zotero.org/styles/religion-in-the-roman-empire" rel="self"/>
```

Comment:
@carljohanberglund, thanks! Did you base this on an existing CSL style? If so, we'd like to know which one, so we can add a "template" link (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link)